lib/svg-sprite/config.js

```diff
@@ -85,6 +85,19 @@ const defaultSVGConfig = {
     precision: -1
 };
 
+/**
+ * @param {any} logger  probable logger
+ * @returns {boolean}   result
+ */
+const isWinstonLogger = logger => {
+    return (
+        isObject(logger) &&
+        logger.level !== undefined &&
+        Array.isArray(logger.transports) &&
```

Comment:
@XhmikosR here is the thing.
`lodash.isObject` returns true if array is passed. Our util is not. So there was an error here too because `transports` is an array

Reply:
Ouch... We might need to revisit our util then.

Reply:
BTW I prefer our isObject because it's tighter, but since we might have broken some stuff, better add a comment and accept Arrays too for now. We should revisit it later if needed.

Reply:
Ok. I started to write tests. The goal is to cover entire project. This will find such errors. Our `isObject` is better, I agree